jx_sqlite/setop_table.py

```diff
@@ -183,7 +183,7 @@ def place(parent_doc_details):
                                     sql_selects.append(unsorted_sql + " AS " + column_alias)
                                     index_to_column[column_number] = nested_doc_details['index_to_column'][column_number] = ColumnMapping(
                                         push_name=s.name,
-                                        push_column_name=s.name.replace("\\.", "."),
```

Comment:
This is ok, it is dealing with formatting the output, assuming it is right, given our changes to the tests and to mo-dots